src/Button.test.js

```diff
@@ -0,0 +1,34 @@
+import Button, { StyledButton } from "./Button";
+import { configure, shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import React from "react";
+
```

Comment:
Try importing 'jest-styled-components' and see how the snapshots look (it actually shows the css in the snapshot so if someone changes the styles the snapshot will tell you). 
https://github.com/styled-components/jest-styled-components
If you like it keep it, if you don't, remove it.